rac_aspace/serializers.py

```diff
@@ -1,60 +1,61 @@
 import argparse
-from configparser import ConfigParser
-# import csv
-import os
+import csv
 
 
-config = ConfigParser()
-config.read("local_settings.cfg")
+class Serializer:
 
-file_path = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), config.get(
-        "Destinations", "filename")
-)
+    def __init__(self, filename, delimiter, data):
+        self.filename = filename
+        self.fieldnames = list(data.keys())
+        self.data = data
+        self.delimiter = delimiter
 
-FILE_TYPE_CHOICES = ["csv", "tsv"]
+    def run(self):
+        with open(self.filename, 'w') as f:
+            writer = csv.DictWriter(
+                f, fieldnames=fieldnames, delimiter=delimiter)
+            writer.writeheader()
+            writer.writerows(data)
 
-column_headings = []
 
-column = 'test'
-heading = 'data'
+class CSVWriter(Serializer):
 
-data = [column, heading]
+    def delimiter(self):
+        delimiter = "','"
+        return delimiter
```

Comment:
```suggestion
    delimiter = "','"
```
You can just set this property directly on this class.